app/src/main/java/com/rokt/roktdemo/ui/demo/DemoHome.kt

```diff
@@ -0,0 +1,108 @@
+package com.rokt.roktdemo.ui.demo
+
+import androidx.compose.foundation.Image
```

Comment:
FYI @danial-rokt you've already reviewed this part before, I just pulled it out of "DemoPage" and moved it here

Reply:
thanks for the heads up